jumanji/environments/packing/bin_pack/reward.py

```diff
@@ -112,7 +121,13 @@ def __call__(
     ) -> float:
         del next_state, is_done
         state.items = cast(ValuedItem, state.items)
-        _, item_id = action
+        if len(action) == 2:
+            _, item_id = action
+            chosen_item_value = tree_slice(state.items, item_id).value
+        elif len(action) == 3:
+            orientation, _, item_id = action
+            chosen_item_value = tree_slice(state.items, (orientation, item_id)).value
+
         chosen_item_value = tree_slice(state.items, item_id).value
```

Comment:
I think this line needs to be removed? Are there not tests that fail due to this?

Reply:
You mean line 131 ? I removed it.